src/wp-includes/block-supports/typography.php

```diff
@@ -263,10 +263,24 @@ function wp_typography_get_preset_inline_style_value( $style_value, $css_propert
  *                    `null` on failure.
  */
 function wp_get_typography_value_and_unit( $raw_value, $options = array() ) {
+	if ( ! is_string( $raw_value ) && ! is_int( $raw_value ) ) {
```

Comment:
For backwards compatibility think we need to support numbers here, that is, `10` and `10.2`